### PR TITLE
Add Coral EdgeTPU compatibility. 

### PR DESCRIPTION
**Pull request by mtyka**
Adapt minigo python engine to use EdgeTPU by adding a custom dual_net.py

This supports both the EdgeTPU dev board and the USB form factor for
Linux and Raspberry Pi hosts. In order not to use dependencies that
aren't present or available on all plattforms the imports are inside
the if statement. Potentially this could be moved to it's own
dual_net_chooser.py or something.

Also added some convenience scripts in minigui/edgetpu

### CODE REVIEW

File: dual_net_edge_tpu.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 Google LLC
```
Comment (tommadams) [inline dual_net_edge_tpu.py]: 2019 :)

File: dual_net_edge_tpu.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+It can only be used for inference and requires a specially quantized and
+compiled model file. For more information see https://coral.withgoogle.com
+"""
+
+import numpy as np
+import features as features_lib
+from edgetpu.basic.basic_engine import BasicEngine
+
+class DualNetworkEdgeTpu(object):
+  def __init__(self, save_file, board_size=19):
```
Comment (tommadams) [inline dual_net_edge_tpu.py]: Please use the `go.N` global variable for the board size, which is controlled by the `BOARD_SIZE` environment variable (and defaults to 19):
https://github.com/tensorflow/minigo/blob/6694df26917ca54173878ff1eeaa97c5e40d9265/go.py#L31

**Comment by tommadams** [comment]
Looks like there are a few lint errors too

**Comment by iopq** [comment]
What kind of performance could I expect out of it? I don't own one yet, but if this gets merged I'd be interested in some numbers

**Comment by tommadams** [comment]
I've only trained a small test model so far so I can't give you any accurate numbers. I'm going to train a bigger model next week and we'll post performance numbers when we have them.

**Comment by tommadams** [comment]
Awesome, thanks!

**Pull request merged**